fe/fe-core/src/main/java/com/starrocks/connector/iceberg/IcebergCatalogType.java

```diff
@@ -20,6 +20,7 @@ public enum IcebergCatalogType {
     CUSTOM_CATALOG,
     GLUE_CATALOG,
     REST_CATALOG,
+    HADOOP_CATALOG,
```

Comment:
Can you explain how do you choose this name? If other system use this name already, you can put the reference in the commit message.

Reply:
The name is defined by iceberg itself. https://iceberg.apache.org/javadoc/latest/org/apache/iceberg/hadoop/HadoopCatalog.html